File: entry.js
requirejs.config({
    paths: {
        kotlin: 'prg/kotlin/1.3.41/kotlin',
        'kotlinx-html-js': 'prg/kotlinx-html-js/0.6.12/kotlinx-html-js',
        mswp: 'prg/mswp',
    'fontawesome' : 'prg/fontawesome-free/js/fontawesome',
    'fontawesome-solid' : 'prg/fontawesome-free/js/solid',
        webfontloader: 'https://ajax.googleapis.com/ajax/libs/webfont/1/webfont'
    }
});

requirejs(['mswp', 'fontawesome', 'fontawesome-solid'], 
function(mswpLib, fa, fas) {
    mswpLib.net.ocsoft.mswp.Context.mainPage.run(mswpSettings);    
});

File: src/jsMain/kotlin/net/ocsoft/mswp/Entry.kt
package  net.ocsoft.mswp

import kotlinx.browser.window
import org.w3c.dom.get

/**
 * run the program
 */
actual fun run() {
    val settings = window.get("mswpSettings")
    Context.mainPage?.run(settings)
}

// vi: se ts=4 sw=4 et:
